packages/@dcl/inspector/src/components/ProjectAssetExplorer/utils.ts

```diff
@@ -29,6 +30,8 @@ export function buildAssetTree(paths: string[]): AssetNodeFolder {
               ? 'audio'
               : lowerPath.endsWith('.mp4')
               ? 'video'
+              : lowerPath.endsWith('.jpg') || lowerPath.endsWith('.jpeg') || lowerPath.endsWith('.png')
```

Comment:
wdy about having an array with valid extensions? something like this:

`const validImageExtension = ['.jpg', '.jpeg', '.png']`

```suggestion
              : validImageExtension.some(ext => lowerPath.endsWith(ext))
```